iris/tests/download_manager/open_download_folder.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from iris.api.helpers.download_manager_utils import download_file, DownloadFiles, downloads_cleanup
```

Comment:
Please remove these imports from test case. Roxana's branch has just landed to dev so from now on we won't do the imports in test cases.